subsys/usb/device_next/class/usbd_cdc_ncm.c

```diff
@@ -231,7 +231,8 @@ struct cdc_ncm_eth_data {
 	uint16_t rx_seq;
 
 	struct k_sem sync_sem;
-	struct k_sem notif_sem;
```

Comment:
I do not think it is worth keeping my commit :-). My suggestion is to squash the last two commits with the first.